StellarDotnetSdk/Responses/SorobanRpc/SimulateTransactionResponse.cs

```diff
@@ -114,73 +116,84 @@ public class SimulateTransactionResponse
     ///     containing the
     ///     <c>RestoreFootprint</c> operation.
     /// </summary>
-    [JsonObject]
     public class RestorePreamble
     {
-        [JsonProperty(PropertyName = "transactionData")]
-        private readonly string _transactionData;
+        public RestorePreamble(string transactionData, long minResourceFee)
+        {
+            TransactionData = transactionData;
+            MinResourceFee = minResourceFee;
+        }
+
+        private string TransactionData { get; }
 
         /// <summary>
         ///     Recommended minimum resource fee to add when submitting the <c>RestoreFootprint</c> operation. This fee is to be
         ///     added on
         ///     top of the Stellar network fee.
         /// </summary>
-        [JsonProperty(PropertyName = "minResourceFee")]
-        public readonly long MinResourceFee;
-
-        public RestorePreamble(string transactionData, long minResourceFee)
-        {
-            _transactionData = transactionData;
-            MinResourceFee = minResourceFee;
-        }
+        public long MinResourceFee { get; }
 
         /// <summary>
         ///     The recommended Soroban Transaction Data to use when submitting the <c>RestoreFootprint</c> operation.
         /// </summary>
         public SorobanTransactionData SorobanTransactionData =>
-            SorobanTransactionData.FromXdrBase64(_transactionData); //TODO Unit test
+            SorobanTransactionData.FromXdrBase64(TransactionData); // TODO Unit test
     }
 
     /// <summary>
     ///     Information about the fees expected, instructions used, etc.
     /// </summary>
-    [JsonObject]
     public class SimulateTransactionCost
     {
         /// <summary>
         ///     Number of the total cpu instructions consumed by this transaction.
         /// </summary>
         [JsonProperty(PropertyName = "cpuInsns")]
-        public readonly long CpuInstructions;
+        public long CpuInstructions { get; }
```

Comment:
here the old json property name is not matching the property name is this correct?

Reply:
Yes, it is correct. Isn't "Insns" the short form of **_Ins_**tructio**_ns_**?

Reply:
Yes it is but what I means is that if you put `[JsonProperty(PropertyName = "cpuInsns")]` on top of a property the json parser wil look for this:
```json
{
  ...
  "cpuInsns": "...",
  ...
}
```

But with it removed the parser will look for this:
```json
{
  ...
  "cpuInstructions": "...",
  ...
}
```

This can give problems can you double check if this is correct? Or do we have a miscommunication here?

Reply:
The JsonProperty attribute is still there, I only changed the field to a property.